src/ert/ensemble_evaluator/config.py

```diff
@@ -25,39 +22,41 @@ def get_machine_name() -> str:
 class EvaluatorServerConfig:
     def __init__(
         self,
-        custom_port_range: range | None = None,
+        port_range: tuple[int, int] | None = None,
         use_token: bool = True,
-        custom_host: str | None = None,
+        host: str | None = None,
         use_ipc_protocol: bool = True,
     ) -> None:
-        self.host: str | None = None
+        self.host: str | None = host
         self.router_port: int | None = None
-        self.url = f"ipc:///tmp/socket-{uuid.uuid4().hex[:8]}"
         self.token: str | None = None
-        self._socket_handle: socket.socket | None = None
-
         self.server_public_key: bytes | None = None
         self.server_secret_key: bytes | None = None
-        if not use_ipc_protocol:
-            self._socket_handle = find_available_socket(
-                custom_range=custom_port_range,
-                custom_host=custom_host,
-                will_close_then_reopen_socket=True,
-            )
-            self.host, self.router_port = self._socket_handle.getsockname()
-            self.url = f"tcp://{self.host}:{self.router_port}"
+        self.use_ipc_protocol: bool = use_ipc_protocol
+
+        if port_range is None:
+            port_range = (51820, 51840 + 1)
```

Comment:
Since this is the default and tuple is immutable you can just do `port_range: tuple[int, int] = (51820, 51840+1)` in the parameter list.